src/components/page-deployment/deployment-overview.js

```diff
@@ -44,16 +44,11 @@ export class DeploymentOverview extends React.Component {
     }
 
     return (
-      <React.Fragment>
+      <div className="o-layout-constrained">
```

Comment:
<div> with class o-layout-contrained should be removed. PageDeployment is a child to PageApplication, and PageApplication wraps all sub-components in a <div> with this class.